docs/api.md

```diff
@@ -91,6 +91,51 @@ Specify `redirectTo` to set the path to redirect to after logging out. If this i
 <LogoutRoute redirectTo='/pathToRedirectTo' />
 ```
 
+## Contexts
+
+#### authenticated (bool)
+
+If you want to know if there's an authenticated account, then include the `authenticated` context.
```

Comment:
Don't you mean *"If you want to know if the user is authenticated or not, include the `authenticated` context."*?